www/bootstrap-tour/.editorconfig

```diff
@@ -0,0 +1,21 @@
+EditorConfig helps developers define and maintain consistent
```

Comment:
Add a `# `.

Perhaps `.editorconfig` should be added to `.gitignore`? Not everyone will use the same settings.